src/cli.js

```diff
@@ -27,16 +28,25 @@ const start = () => {
     })
     .help().argv
 
-  const filePath = findFile('.esprintrc');
+  const commands = yargs.argv._;
```

Comment:
There's a way to set up commands using yargs: http://yargs.js.org/docs/#methods-commandmodule
Might simplify things a little